canonical/ubuntu.com
# Add CVEs About page

## Done

- Add CVEs about page
- Copy docs: [link](https://docs.google.com/document/d/1fsKWdmAEv4QEoVPsZEwff3fYG3a6RkD5-BCkk13LxPQ/edit#heading=h.dcxj13sf7o5g)
- Figma: [link](https://www.figma.com/design/nUJEBpOPTW74RqDRuMQXxB/23.04-U.com---CVEs?node-id=949-23441&m=dev)
- Refactor `/cves` to `/cve` and apply djlint formatting on all the files

## QA

- Go to https://ubuntu-com-14019.demos.haus/security/cves/about
- Check that design and details adheres to Figma and copy docs
- Check that side nav works
  - It should jump to heading on select
  - When scrolling through headings, the side nav links will update automatically

## Issue / Card

Fixes [WD-11852](https://warthogs.atlassian.net/browse/WD-11852)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-11852]: https://warthogs.atlassian.net/browse/WD-11852?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
